src/foam/u2/view/RichChoiceView.js

```diff
@@ -325,6 +325,7 @@ foam.CLASS({
                     .select(section.filtered || section.dao, (obj) => {
                       return this.E()
                         .start(self.rowView, { data: obj })
+                          .attrs({ name: (obj.name).replace(/[^\w\d]/g,'') })
```

Comment:
Is the the proper use of the 'name' attribute? Why are we doing this?

Reply:
believe this should be 
`obj.cls_.id`

Reply:
Not needed. Troy will set the name and the value on the top-level <div>.
FYI: @TW80000